akka-projection-cassandra/src/main/scala/akka/projection/cassandra/scaladsl/CassandraProjection.scala

```diff
@@ -28,6 +25,10 @@ import akka.projection.scaladsl.Handler
 import akka.projection.scaladsl.SourceProvider
 import akka.stream.scaladsl.FlowWithContext
 
+import scala.collection.immutable
+import scala.concurrent.Future
+import scala.concurrent.duration.Duration
```

Comment:
we have agreed on intellij sorting settings as it was, to avoid merge conflicts

Reply:
Hmm, I'll look around but I'm not sure I know all the knobs that need tuning. :-(